content/apprenticeships/barclays.md

```diff
@@ -0,0 +1,8 @@
+---
+company: "Barclays"
+description: "Kickstart your career in Technology with a Foundation or Higher Apprenticeship."
+image: "barclays.jpg"
+link: "https://joinus.barclays/eme/apprenticeships/"
+location:
+  - "London, Northampton or Radbroke Hall in Knutsford"
```

Comment:
For parsing reasons, can you reformat this like the other files?
```md
location:
  - "San Francisco, CA"
  - "Salt Lake City, UT"
  - "Austin, TX"
```

Reply:
@benjaminmodayil changed